packages/@misk/core/src/features/Navbar/DimensionAwareNavbar.tsx

```diff
@@ -11,6 +11,7 @@ import {
   IDimensionAwareProps,
   IMenuExternalProps,
   Menu,
+  MiskNavbarGroup,
```

Comment:
Let's name this `NavbarGroup` since everything in the `Navbar` package used to be prefixed with `Misk` until I realized that it's clear from the import where something is coming from so using the same names as BlueprintJS is not bad.